src/HospitalLibrary/Auth/AuthService.cs

```diff
@@ -34,15 +35,27 @@ public async Task<List<Allergen>> GetPatientsAllergens(List<AllergenDTO> allerge
 
             return allergens;
         }
-        public async Task<Users.User> RegisterPatient(Users.User user,List<AllergenDTO> allergens)
+        private async Task<Doctor> GetPatientsDoctor(string doctorUid)
+        {
+            foreach (Doctor doctor in await _unitOfWork.DoctorRepository
+                         .GetTwoIternalMedicineDoctorsAscendingByPatientNumber())
```

Comment:
Move repository call to the separate line, its more readable. Maybe also refactor method name.